lib/shared/cross_posts.dart

```diff
@@ -10,6 +13,8 @@ import 'package:thunder/utils/instance.dart';
 import 'package:thunder/post/utils/navigate_create_post.dart';
 import 'package:thunder/post/utils/navigate_post.dart';
 
+import '../community/widgets/post_card_metadata.dart';

```

Comment:
Use absolute imports like the other imports for conventions!

Reply:
This one was probably automatically added by android studio.